ai-eval-service/evaluator.py

```
@@ -361,18 +251,22 @@ async def evaluate_batch(
-            "ethnicity": "race",
-            "religion": "religion",
-            "age": "age",
-        }
-        
         stereotype_results = []
         category_groups = {}
         for item in validated_items:
-            langfair_category = category_mapping.get(item['category'].lower(), "gender")
+            category_lower = item['category'].lower()
+            if category_lower not in self.CATEGORY_MAPPING:
+                frame = inspect.currentframe()
```

Ruff (0.14.10)

Undefined name `inspect`

(F821)